frontend/src/container/Home.jsx

```diff
@@ -86,6 +207,77 @@ export class Home extends React.Component {
                 )}
               </CardBody>
             </Card>
+            {(this.state.usernamePopup &&
+              <CardPopup
+                customClass='homepage__usernamePopup'
+                customHeaderClass='primaryColorBg'
+                onClose={this.handleClickCloseUsernamePopup}
+              >
+                <div className='homepage__usernamePopup__body'>
+                  <div className='homepage__usernamePopup__body__title'>
+                    {props.t("Hello, you don't have a username yet!")}
+                  </div>
+
+                  <div className='homepage__usernamePopup__body__msg'>
+                    {props.t('Set your username:')}
+                  </div>
+
+                  <input
+                    className='homepage__usernamePopup__body__input form-control'
+                    type='text'
+                    placeholder={props.t('@username')}
+                    value={this.state.newUsername}
+                    onChange={this.handleChangeNewUsername}
+                    data-cy='usernamePopup_username'
+                  />
+
+                  {!this.state.newUsernameAvailability && (
+                    <div className='homepage__usernamePopup__errorMsg'>
+                      <i className='homepage__usernamePopup__errorIcon fa fa-times' />
+                      {props.t('This username is not available')}
+                    </div>
+                  )}
+
+                  {this.state.newUsername !== '' && (
+                    <>
+                      <div className='homepage__usernamePopup__body__msg'>
+                        {props.t('Please confirm your password:')}
+                      </div>
+
+                      <input
+                        className='homepage__usernamePopup__body__input form-control'
+                        type='password'
+                        placeholder={props.t('Password')}
+                        value={this.state.password}
+                        onChange={this.handleChangePassword}
+                        data-cy='usernamePopup_password'
+                      />
+                    </>
+                  )}
+
+                  <div className='homepage__usernamePopup__body__checkbox'>
+                    <input
+                      className='homepage__usernamePopup__body__checkbox__input'
+                      type='checkbox'
+                      onChange={this.handleClickCheckbox}
+                    />
+                    {props.t('never ask me again')}
```

Comment:
Should this be capitalized?

Reply:
Personally I prefer it that way, but I was in doubt, maybe you're right, @celiamgtt what do you think?

Reply:
Yes it would be better ;)

Reply:
Changed.